package.json

```diff
@@ -69,6 +69,21 @@
 						"simple"
 					]
 				},
+				"fileAccess.hideWhenWritable": {
+					"type": "boolean",
+					"default": false,
+					"description": "Hide the Status Bar indicator when the file is Writable"
+				},
+				"fileAccess.readonlyColor": {
```

Comment:
Use `contributes.color` instead, as described in https://code.visualstudio.com/api/references/contribution-points#contributes.colors

Reply:
It's great that theme makers can override the colors.  But how are end users meant to find out about the colors and edit them?  It doesn't seem very user-friendly (not that editing colors as numbers with no preview is very user-friendly, but at least it's discoverable and simple.